concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -881,7 +881,7 @@ Hale-1973-1798-879	Food_and_Clothing-76	overflow	v	07b1.26
 Hale-1973-1798-880	Food_and_Clothing-77	spill	v	07b1.27	3130	SPILL (SOMETHING)
 Hale-1973-1798-881	Food_and_Clothing-78	become empty	v	07b1.28		
 Hale-1973-1798-882	Food_and_Clothing-79	pour	v	07b1.29	1487	POUR
-Hale-1973-1798-883	Food_and_Clothing-80	bubble	v	07b1.30		
+Hale-1973-1798-883	Food_and_Clothing-80	bubble	v	07b1.30	3689	BUBBLE
```

Comment:
`3689 BUBBLE` is a noun in concepticon: https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L3675